packages/ui/src/atoms/Slider/Slider.tsx

```diff
@@ -23,6 +27,10 @@ export type SliderProps = {
    * Callback that fires when the slider value changes.
    */
   onChange?: (value: { min: number; max: number }) => void
+  /**
+   * Callback that fires when the slider value ends changing.
+   */
+  onEnd?: (value: { min: number; max: number }) => void
```

Comment:
this has the same behavior as `onBlur`, why not call it that then?

Reply:
not really, on end fires when the user stops clicking the element. `onBlur` fires when the object it blurred, i.e. the user clicks, interacts and them leaves.